lusidtools/cocoon/utilities.py

```diff
@@ -1354,8 +1355,26 @@ def create_scope_id(time_generator=None):
     # Multiply by 7 to get value to 100s of nano seconds
     timestamp = hex(int(test * 10000000.0))
```

Comment:
Perhaps change the `test` variable name to `current_time`

Reply:
Changed